src/main/java/hudson/plugins/audit_trail/AuditTrailFilter.java

```diff
@@ -34,21 +39,33 @@
 import java.util.regex.Pattern;
 import java.util.regex.PatternSyntaxException;
 
+import static hudson.init.InitMilestone.PLUGINS_PREPARED;
 
 /**
  * Servlet filter to watch requests and log those we are interested in.
  * @author Alan Harder
+ * @author Pierre Beitz
  */
+@Extension
 public class AuditTrailFilter implements Filter {
 
     private static final Logger LOGGER = Logger.getLogger(AuditTrailFilter.class.getName());
 
     private static Pattern uriPattern = null;
```

Comment:
As you store the pattern in String in the config, it's perhaps better to have it compiled to Pattern as well in the config to avoid a static field here. Especially with the static setting.